packages/shared/src/components/filters/FeedFilters.tsx

```diff
@@ -12,46 +20,63 @@ interface FeedFiltersProps {
   onBack?: () => void;
 }
 
-let timeout: NodeJS.Timeout;
-
 export default function FeedFilters({
   isOpen,
   onBack,
 }: FeedFiltersProps): ReactElement {
+  const { alerts, updateAlerts } = useContext(AlertContext);
+  const { feedSettings } = useFeedSettings();
   const [hidden, setHidden] = useState(true);
+  const timeoutRef = useRef<number>();
 
   useEffect(() => {
     if (isOpen) {
-      if (timeout) clearTimeout(timeout);
+      if (
+        alerts?.filter &&
+        (feedSettings?.includeTags?.length ||
+          feedSettings?.blockedTags?.length ||
+          feedSettings?.excludeSources?.length ||
+          feedSettings?.advancedSettings?.length)
+      ) {
+        updateAlerts({ filter: false });
+      }
```

Comment:
Don't have this update statement in another component or a hook as well?

Reply:
We don't but I think it would be a good idea to simplify this. Let me commit one thing.